framework/BaseYii.php

```diff
@@ -210,11 +211,14 @@ public static function getRootAlias($alias)
      * @throws InvalidParamException if $path is an invalid alias.
      * @see getAlias()
      */
-    public static function setAlias($alias, $path)
+    public static function setAlias($alias, $path, $overwrite = true)
```

Comment:
I don't think we need it as option. It should be default behavior not to overwrite it.